### PR TITLE
GPC-NONE: Initial SAM deploy

### PR DESCRIPTION
**Pull request by oskarwilliams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,26 @@ Restart terminal then run in the project top directory
 sam build
 ```
 
+### Deploying
+
+To deploy our SAM to the dev environment, please create your own configuration and then apply with that. To create your own configuration, modify and add the following to `samconfig.toml`
+```
```
Comment (olevett) [inline README.md:66]:
```suggestion

```
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,26 @@ Restart terminal then run in the project top directory
...
+resolve_s3 = true
+s3_prefix = "<your-name>-lev"
+region = "eu-west-2"
+confirm_changeset = true
+capabilities = "CAPABILITY_IAM"
+parameter_overrides = "Environment=\"dev-<your-name>\""
+```
+
+Once you have added this, you can then run the below while logged into the dev environment via SSO to deploy to dev:
+```shell
```
Comment (olevett) [inline README.md:78]:
```suggestion

```shell
```

File: README.md (hunk 1 above)
Comment (olevett) [inline README.md:66]:
```suggestion

```ini
```

File: samconfig.toml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+version = 0.1
+
+[dev-oskwil.deploy.parameters]
+stack_name = "oskwil-lev"
```
Comment (olevett) [inline samconfig.toml:4]: Hmm - should this be committed or ignored?

**Review by olevett — CHANGES_REQUESTED** [review]
Couple of comments around, hopefully nothing major though.

File: samconfig.toml (hunk 3 above)
Comment (oskarwilliams) [reply samconfig.toml:4]: committed, we want to track this

File: template.yaml [hunk 4]
```diff
@@ -96,19 +83,87 @@ Resources:
     Properties:
       KmsMasterKeyId: !Ref MasterKmsKey
 
+  DeathEnrichmentDlq:
```
[the remark this answers is not in the document]
Comment (oskarwilliams) [reply template.yaml:86]: see the adr, it has everything except the dlqs, and this has all of the drawing except the api gateway and earlier

**Review by olevett — APPROVED** [review]

**Pull request merged**